### PR TITLE
Group 4 - W2 commands solution

### PR DESCRIPTION
**Pull request by jangeador**
Group 4:
Jason Symons
Delio Castillo

### CODE REVIEW

File: cmd_line_tools/commands.py [hunk 1]
```diff
@@ -1,11 +1,195 @@
...
+
+
+class PokemonBerryPrinter(PagedJSONDataMixin,
+                          StdoutOutputMixin,
+                          BaseAllPokemonBerriesCommand):
+    """ Prints to stdout """
+
+    def main(self):
+        results = self.get_pokemon()
+        for berry in results:
```
Comment (Psy-Kosh) [inline cmd_line_tools/commands.py:87]: My pokeknowledge is rather limited, though I like the idea of your project. Am just curious, though... "berry"?

File: cmd_line_tools/commands.py [hunk 2]
```diff
@@ -1,11 +1,195 @@
...
+
+
+class RandomChooserPokemon(JSONDataRequestMixin,
+                           StdoutOutputMixin,
+                           BasePokemonCommand):
+    """
+    Retrieves information of random pokemon
+    N_POKEMON = Number of pokemon available from PokeAPI
+    """
+    N_POKEMON = 811
```
Comment (Psy-Kosh) [inline cmd_line_tools/commands.py:175]: Seems would have been better to get this value from the API rather than hard coding it.

**Review by Psy-Kosh — COMMENTED** [review]
I didn't have too much to comment this time, but I will say impressive, looks like you two did a fairly comprehensive project here.

File: cmd_line_tools/commands.py (hunk 1 above)
Comment (jangeador) [reply cmd_line_tools/commands.py:87]: Yes, pokemon eat berries. there are several distinct berry types.

Comment (Psy-Kosh) [reply cmd_line_tools/commands.py:87]: Ah, okay, thanks.

**Pull request closed without merging**